src/cfml/system/util/CompileDSL.cfc

```diff
@@ -0,0 +1,88 @@
+
+/**
+*********************************************************************************
+* Copyright Since 2014 CommandBox by Ortus Solutions, Corp
+* www.coldbox.org | www.ortussolutions.com
+********************************************************************************
+* @file  D:\Dev\Repos\commandbox\src\cfml\system\util\CompileDSL.cfc
```

Comment:
There's really no need for the file or author in the days of source control.  The description is great tho.

Reply:
awesome i removed them, they will not appear in the next commits